### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by IvanSidorenkov**

### CODE REVIEW

File: src/sections/calculator/Calculator.tsx [hunk 1]
```diff
@@ -15,36 +13,51 @@ interface IVat {
...
 
   const handleActiveVat = (vat: string) => setActiveVat(vat);
   const onMouseEnter = () => setIsHovered(true);
   const onMouseLeave = () => setIsHovered(false);
-  const handlePrice = (value: React.ChangeEvent<HTMLInputElement>) => setPrice(Number(value.target.value));
+  const handlePrice = (value: React.ChangeEvent<HTMLInputElement>) => {
+    const num = value.target.value;
+
+    value.target.value = String(num.length > 8 ? 99999999 : Number(value.target.value));
+    setPrice(num.length > 8 ? 99999999 : Number(value.target.value));
```
Comment (AndreiLavrov) [inline src/sections/calculator/Calculator.tsx:54]: I'm not sure about logic here, is it ok? maybe use vars and cut a num after 8 digets?

**Review by AndreiLavrov — APPROVED** [review]

File: src/components/shop/Shop.tsx [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
-      <div className={styles.labelWrapper}>
-        <label className={styles.label}>{label}</label>
-      </div>
-      <div className={styles.imageWrapper}>
-        <Image src={image || mockedImage} alt="shop" height={50} width={147} />
-      </div>
-    </div>
+    <li>
+      <Link href={link} className={styles.wrapper} target="_blank">
+        <div className={styles.labelWrapper}>
```
Comment (ssavitski) [inline src/components/shop/Shop.tsx:17]: figcaption?

File: src/constants/constants.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+export const TAGS_TEXT = [
+  {
+    id: 1,
+    upText: '5 лет',
+    downText: 'привозим товары',
+  },
+  {
+    id: 2,
```
Comment (ssavitski) [inline src/constants/constants.ts:8]: what for do we need such objects?

File: src/layout/footer/Footer.module.scss [hunk 4]
```diff
@@ -51,11 +55,26 @@
   column-gap: 120px;
 }
 
+.houseImage {
```
Comment (ssavitski) [inline src/layout/footer/Footer.module.scss:58]: we can add one class with similar stylization for all elements, not 4 classes with the same style or styles

File: src/sections/calculator/Calculator.tsx [hunk 5]
```diff
@@ -55,73 +70,75 @@ const Calculator = () => {
...
-              </div>}
-              <Image
-                src={infoIcon}
-                alt="info icon"
-                className={styles.tooltip}
-                onMouseLeave={onMouseLeave}
-                onMouseEnter={onMouseEnter}
-              />
+    <section id="calculator" className={styles.border}>
+      <div className={styles.borderLeft} />
```
Comment (ssavitski) [inline src/sections/calculator/Calculator.tsx:74]: what for do we need it?

**Review by ssavitski — APPROVED** [review]

**Pull request merged**